deploy/kube-config/kubedash/heapster-controller.json

```diff
@@ -0,0 +1,40 @@
+{
+    "apiVersion": "v1",
+    "kind": "ReplicationController",
+    "metadata": {
+	"labels": {
+	    "name": "heapster"
+	},
+	"name": "monitoring-heapster"
+    },
+    "spec": {
+	"replicas": 1,
+	"selector": {
+	    "name": "heapster"
+	},
+	"template": {
+	    "metadata": {
+		"labels": {
+		    "name": "heapster"
+		}
+	    },
+	    "spec": {
+		"containers": [
+		    {
+			"image": "afein/heapster-model:latest",
```

Comment:
That's a fair point. I no longer have access to a gcr.io bucket so someone else would have to actually deploy this.

These deployment files can be eventually deprecated, as soon as the next milestone is deployed in all kubernetes clusters. In that case, the model should be accessible by default in the heapster pod running in the kube-system namespace.